src/Enzo/enzo_EnzoMethodPmDeposit.cpp

```diff
@@ -180,6 +208,9 @@ void EnzoMethodPmDeposit::compute ( Block * block) throw()
 
           for (int ip=0; ip<np; ip++) {
 
+	    // Stefan: Unsure about this line, doesn't match what's in the
+	    // Enzo paper, which says that particles are drifted by half a timestep
+	    // before being deposited to the grid
             double x = xa[ip*dp] + vxa[ip*dv]*dt;
```

Comment:
In the light of the comment above by Stefan, this needs to be looked at ASAP before approving, preferrably by someone who is the expert at evolving 'Particles'.